samples/Catalog/android/app/build.gradle

```diff
@@ -115,25 +115,28 @@ android {
         variant.outputs.each { output ->
```

Comment:
Splitting the APK is not useful for our case,  the `build.gradle` file is generated by React and is used as a general template for the native module.

The Catalog app embedded in the repo in `samples/Catalog` should mimic a React Native project generated by `react-native init Catalog` with the  `react-native-pspdfkit` module linked.